core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -1023,12 +1024,41 @@ private static Type parameterizedTypeOf(AnnotatedElement element) {
     }
 
     static boolean isAnnotatedNullable(AnnotatedElement annotatedElement) {
+        // 1) declaration annotation
```

Comment:
Could you add examples so that readers can distinguish them easily? e.g.
```
@Nullable // declaration annotation
public String declarationAnnotatedMethod() {
    return null;
}

// type-use annotation
public @Nullable String typeUseAnnotatedMethod() {
    return null;
}
```

Reply:
Ok, I added comment
[`ca1e429` (#6457)](https://github.com/line/armeria/pull/6457/commits/ca1e429a8a566d9ba8c174f130b55c17807ae254)